lib/src/components/__tests__/SimpleComponents.test.tsx

```diff
@@ -343,3 +344,9 @@ describe('NxSystemNotice.Container', function() {
     expect(shallow(<NxSystemNotice.Container/>)).toMatchSelector('div.nx-system-notice-container');
   });
 });
+
+describe('NxDivider', function() {
+  it('makes a <hr> tag with an nx-divider class', function() {
```

Comment:
```suggestion
  it('makes a <hr> tag with a nx-divider class', function() {
```

Reply:
Actually we settled long ago on using "an" in this case (which comes up many many times in the RSC)

Reply:
> Actually we settled long ago on using "an" in this case (which comes up many many times in the RSC)

Ah thanks! Good to know 👍